redbeams-api/src/main/java/com/sequenceiq/redbeams/doc/OperationDescriptions.java

```diff
@@ -5,10 +5,12 @@
     public static final class DatabaseOpDescription {
         public static final String TEST_CONNECTION = "test database connectivity";
         public static final String LIST = "list database configs";
-        public static final String GET_BY_NAME = "get database config by name";
+        public static final String GET_BY_NAME = "get a database config by name";
+        public static final String GET_BY_CRN = "get a database config by CRN";
         public static final String REGISTER = "register database config of existing database";
```

Comment:
NIT: All other descriptions use articles.

Reply:
Making them all consistent, using articles. Also fixing verb forms in server CREATE and TERMINATE descriptions